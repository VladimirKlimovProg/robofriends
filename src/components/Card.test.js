import { shallow } from 'enzyme';
import React from 'react'
import Card from './Card';

it('expect to render Card componet', () => {
    expect(shallow(<Card />)).toMatchSnapshot();
})

