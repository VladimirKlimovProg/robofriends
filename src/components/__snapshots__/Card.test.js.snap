// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card componet 1`] = `
<div
  className="tc bg-light-green br3 dib pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt="robots"
    src="https://robohash.org/undefined?size=200x200"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
